import-export-cli/impl/importThrottlePolicy.go

```diff
@@ -41,7 +43,15 @@ func ImportThrottlingPolicyToEnv(accessOAuthToken, importEnvironment, importPath
 }
 
 func importThrottlingPolicy(endpoint string, importPath string, accessToken string, isOauth bool, ThrottlePolicyUpdate bool) error {
-	resp, err := executeThrottlingPolicyUploadRequest(endpoint, importPath, ThrottlePolicyUpdate, accessToken, isOauth)
+	exportDirectory := filepath.Join(utils.ExportDirectory, utils.ExportedPoliciesDirName, utils.ExportedThrottlePoliciesDirName)
+	resolvedPolicyFilePath, err := resolvePolicyImportFilePath(importPath, exportDirectory)
+	if err != nil {
+		return err
+	}
+
+	utils.Logln(utils.LogPrefixInfo+"Policy Location:", resolvedPolicyFilePath)
+
+	resp, err := executeThrottlingPolicyUploadRequest(endpoint, resolvedPolicyFilePath, ThrottlePolicyUpdate, accessToken, isOauth)
 	utils.Logf("Response : %v", resp)
 	if err != nil {
 		utils.Logln(utils.LogPrefixError, err)
```

Comment:
Shouldn't we first check if there is an error and log? Looks like we log the response `resp` before we check if there is an error. Or is this fine?